src/VisualStudio/CSharp/Test/F1Help/F1HelpTests.cs

```diff
@@ -798,6 +798,34 @@ void M2()
 }", "System.Int32");
         }
 
+        [Fact]
+        public async Task TestRefReadonlyParameter_Ref()
+        {
+            await TestAsync(
+                """
+                class C
+                {
+                    void M(r[||]ef readonly int x)
```

Comment:
Not related to this PR:
It looks like we currently don't have a page for "ref readonly". Instead, ref readonly returns are covered under the ["readonly" page](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/readonly#ref-readonly-return-example)
@BillWagner, it is possible for us to have F1/help on `ref readonly` go to a dedicated "ref readonly" page. Let us know if that's preferable.